File: trunk/UMessage/src/com/lollotek/umessage/classes/SingleChatMessage.java
package com.lollotek.umessage.classes;

public class SingleChatMessage {

	private String idMessage, idChat, direction, status, data, type, message, read;

	public SingleChatMessage() {
		this.idMessage = "";
		this.idChat = "";
		this.direction = "";
		this.message = "";
		this.status = "";
		this.data = "";
		this.type = "";
		this.read = "";
	}

	public SingleChatMessage(String idMessage, String idChat, String direction,
			String status, String data, String type, String message, String read) {
		super();
		this.idMessage = idMessage;
		this.idChat = idChat;
		this.direction = direction;
		this.status = status;
		this.data = data;
		this.type = type;
		this.message = message;
		this.read = read;;
	}

	public String getIdMessage() {
		return idMessage;
	}

	public void setIdMessage(String idMessage) {
		this.idMessage = idMessage;
	}

	public String getIdChat() {
		return idChat;
	}

	public void setIdChat(String idChat) {
		this.idChat = idChat;
	}

	public String getDirection() {
		return direction;
	}

	public void setDirection(String direction) {
		this.direction = direction;
	}

	public String getStatus() {
		return status;
	}

	public void setStatus(String status) {
		this.status = status;
	}

	public String getData() {
		return data;
	}

	public void setData(String data) {
		this.data = data;
	}

	public String getType() {
		return type;
	}

	public void setType(String type) {
		this.type = type;
	}

	public String getMessage() {
		return message;
	}

	public void setMessage(String message) {
		this.message = message;
	}

	public String getRead() {
		return read;
	}

	public void setRead(String read) {
		this.read = read;
	}

}

File: UMessagePHP/classes/SingleChatMessage.php
<?php

class SingleChatMessage{

	private $id, $idchat, $direction, $msg, $status, $data, $type;
	 
	function __construct($id, $idchat, $direction, $msg, $status, $data, $type){
		$this->id = $id;
		$this->idchat = $idchat;
		$this->direction = $direction;
		$this->msg = $msg;
		$this->status = $status;
		$this->data = $data;
		$this->type = $type;
		
	}

	public function getId()
	{
	    return $this->id;
	}

	public function setId($id)
	{
	    $this->id = $id;
	}

	public function getIdchat()
	{
	    return $this->idchat;
	}

	public function setIdchat($idchat)
	{
	    $this->idchat = $idchat;
	}

	public function getDirection()
	{
	    return $this->direction;
	}

	public function setDirection($direction)
	{
	    $this->direction = $direction;
	}

	public function getMsg()
	{
	    return $this->msg;
	}

	public function setMsg($msg)
	{
	    $this->msg = $msg;
	}

	public function getStatus()
	{
	    return $this->status;
	}

	public function setStatus($status)
	{
	    $this->status = $status;
	}

	public function getData()
	{
	    return $this->data;
	}

	public function setData($data)
	{
	    $this->data = $data;
	}

	public function getType()
	{
	    return $this->type;
	}

	public function setType($type)
	{
	    $this->type = $type;
	}
}

?>